typescript/examples/by-use-case/token-discovery-with-coingecko/index.ts

```diff
@@ -0,0 +1,89 @@
+import { coinGeckoTokenDiscovery } from "@goat-sdk/plugins-coingecko-token-discovery";
+import { evmKeyPair } from "@goat-sdk/wallets-evm";
+import { solanaKeypair } from "@goat-sdk/wallets-solana";
+import { Connection, clusterApiUrl } from "@solana/web3.js";
+import { config } from "dotenv";
+
+config();
+
+const COINGECKO_API_KEY = process.env.COINGECKO_API_KEY;
+
+if (!COINGECKO_API_KEY) {
+    console.error("COINGECKO_API_KEY environment variable is required");
+    process.exit(1);
+}
+
+async function main() {
```

Comment:
Im going to assume this is completely hallucinated haha. Could we create a proper example to test?

Reply:
Have you tried the other two Python examples provided by the PR?